lib/absinthe/schema/notation.ex

```diff
@@ -1401,16 +1420,27 @@ defmodule Absinthe.Schema.Notation do
     scoped_def(env, :enum, identifier, attrs, block)
   end
 
-  defp reformat_description(text), do: String.trim(text)
```

Comment:
Looks like we lost the `String.trim` call on `description`s, I suspect this is something we'll want to get back, right @benwilson512?